### DISCUSSION TITLE
Grafana alerting filter based on matching instances

### DISCUSSION THREAD

**Discussion by pratikdas44** [grafana/grafana#40066] [Alerting]
Hi all, i wanted to filter alerts based on matching instances. So instances only showing alerting should be seen, omitting normal and pending ones. Also, is there any way to achieve it or by using some other tool.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
